tests/Commit.py

```diff
@@ -132,12 +132,12 @@ def testCreateStatusWithAllParameters(self):
         status = self.commit.create_status(
             "success",
             "https://github.com/jacquev6/PyGithub/issues/67",
-            "Status successfuly created by PyGithub",
+            "Status successfully created by PyGithub",
         )
         self.assertEqual(status.id, 277040)
         self.assertEqual(status.state, "success")
         self.assertEqual(status.target_url, "https://github.com/jacquev6/PyGithub/issues/67")
-        self.assertEqual(status.description, "Status successfuly created by PyGithub")
+        self.assertEqual(status.description, "Status successfully created by PyGithub")
```

Comment:
this change requires changing `tests/ReplayData/Commit.testCreateStatusWithAllParameters.txt` as well.